Subnet-Calculator/admin/sw.js

```
@@ -0,0 +1,43 @@
+// is left clean. New admin sessions never register this file, because
+// app.js (v3.2.3+) skips registration on /admin/* paths.
+
+self.addEventListener('install', event => {
+    event.waitUntil(self.skipWaiting());
+});
+
+self.addEventListener('activate', event => {
+    event.waitUntil((async () => {
+        try {
+            await self.registration.unregister();
+            const clients = await self.clients.matchAll({ includeUncontrolled: true });
```

ESLint

[error] 'self' is not defined.

(no-undef)